src/pages/Login.vue

```diff
@@ -214,15 +214,15 @@ export default {
     },
     requestedOtp()
     {
-       if (!this.resendOTPTimer) clearInterval(this.interval);
+       if (!this.requestedOtp) clearInterval(this.otpTimerInterval);

```

Comment:
the condition is not correct here - please don't rush - think through slowly!

Reply:
earlier you said to watch requestOtp when it turns false

Reply:
when we change the input the `requestedOTP` value will be changed to false

Reply:
the watcher will see and will not have to wait

Reply:
a better solution would be to call clearInterval as soon as the requestedOtp value changes to false (based on its watcher) because here there is still a possibility that one might see resend otp = -1 - very slim chance but possible. I made changes according to this.